packages/zk-certificates/contracts/ZkCertificateRegistry.sol

```diff
@@ -100,60 +135,69 @@ contract KYCRecordRegistry is Initializable, IKYCRegistry {
         }
 
         // Set merkle root
-        merkleRoot = currentZero;
+        merkleRoots.push(currentZero);
```

Comment:
we need to put `merkleRootValidIndex` to be at least 1, because the default value of the mapping `merkleRootIndex` is 0, so if `merkleRootValidIndex` starts as 0 then all merkle root values will be valid in the beginning. Also the zero root doesn't have any zkCertificate to be used.